File: app/src/main/java/com/example/bottlerocketstudioscodingtest/Utils/APIInterface.java
package com.example.bottlerocketstudioscodingtest.Utils;

import com.example.bottlerocketstudioscodingtest.Model.StoreList;

import retrofit2.Call;
import retrofit2.http.GET;

public interface APIInterface {
    @GET("stores.json")
    Call<StoreList> getContent();
}

File: app/src/main/java/com/example/bottlerocketstudioscodingtestkotlin/Utils/APIInterface.kt
package com.example.bottlerocketstudioscodingtestkotlin.Utils

import com.example.bottlerocketstudioscodingtestkotlin.Model.StoreList

import retrofit2.Call
import retrofit2.http.GET

interface APIInterface {
    @get:GET("stores.json")
    val content: Call<StoreList>
}
